wiki/Tournaments/id.md

```diff
@@ -1,336 +1,702 @@
----
-outdated: true
----
-
 # Turnamen
 
-Bagian ini diperuntukkan untuk Turnamen osu!. Turnamen memiliki forum khusus yang dapat Anda temukan [melewati tautan berikut](https://osu.ppy.sh/community/forums/55).
+Bagian ini diperuntukkan untuk turnamen osu!. Turnamen memiliki forum khusus yang dapat kamu temukan [melewati tautan ini](https://osu.ppy.sh/community/forums/55).
```

Comment:
feel like this could be better

`Laman ini diperuntukkan untuk turnamen-turnamen yang ada di osu!. Turnamen memiliki forum khusus yang dapat kamu temukan [pada tautan berikut]`